add_to_app/multiple_flutters/multiple_flutters_android/app/src/main/java/dev/flutter/multipleflutters/DoubleFlutterActivity.kt

```diff
@@ -20,6 +20,10 @@ class DoubleFlutterActivity : FragmentActivity(), EngineBindingsDelegate {
         EngineBindings(activity = this, delegate = this, entrypoint = "bottomMain")
     }
     private val numberOfFlutters = 2
+    private var engineCountStart = 0
+    private companion object {
+        var engineCount = 0
```

Comment:
`totalEngineCount` for readability? Since you have a lot of `engine*` names in this file.

Reply:
IMO that is a bit misleading because that would suggest it was the number of engines alive, not the number of engines ever made.

Reply:
you're right. Let's leave it as is.